delivery/media/cat/examples/token-verifier-example-2/main.js

```diff
@@ -0,0 +1,162 @@
+        let result = cat.isCATWellFormed(catJSON.payload);
+        logger.log("result: %o", result)
+        if (result.status === true) {
+          try {
+            // Perform signature verification with expiry and nbf field check
+            await cwtValidator.validate(catTokenBuf,[{ key: verificationKey }]);
+            // Check is CAT claim set is acceptable. 
+            result = await cat.isCATAcceptable(catJSON.payload, request);
+            logger.log("result: %o", result)
+            if (result.status === true) {
+              // // check renewal
+              // const catr = catJSON.payload.get(ClaimsLabelMap.catr);
+              // const exp = catJSON.payload.get(ClaimsLabelMap.exp);
+              // const renewalType = catr.get(CatRLabelMap.renewal_type);
+              // const exp_extension = catr.get(CatRLabelMap.exp_extension);
+              // const exp_deadline = catr.get(CatRLabelMap.exp_deadline);
+              // logger.log("renewalType: %s", renewalType)
+              // logger.log("exp_extension: %s", exp_extension)
+              // logger.log("exp_deadline: %s", exp_deadline)
+              // logger.log("exp: %s", exp)
+              // // support for cookie and header renewable type
+              // if ((renewalType === 1 || renewalType === 2)  && exp_extension !== undefined && exp !== undefined) {
+              //   let lowT;
+              //   if (exp_deadline !== undefined) {
+              //     lowT = exp - exp_deadline * 60
+              //   } else {
+              //     lowT = exp - 1 * 60 // 1 mins renewal window by default
+              //   }
+              //   const now = Math.floor(Date.now()/1000);
+              //   logger.log("lowT: %s", lowT)
+              //   logger.log("now: %s", now)
+              //   // renew the token
+              //   if (now >= lowT && now < exp) {
+              //     const new_exp = now + exp_extension
+              //     catJSON.payload.set(ClaimsLabelMap.exp, new_exp)
+              //     catJSON.payload.set(ClaimsLabelMap.iss, 'akamai.com')
+              //     catJSON.payload.set(ClaimsLabelMap.iat, now)
+              //     // renewal token is signed with same key
+              //     catJSON.header.u.set(HeaderLabelMap.kid, new TextEncoder().encode('akamai_key_hs256'))
+              //     catJSON.header.p.set(HeaderLabelMap.alg, AlgoLabelMap.HS256);
+              //     const cwtTokenBuf = await CWTGenerator.sign(catJSON.payload, { key: verificationKey }, catJSON.header, { isCoseCborTagAdded: true, isCWTTagAdded: true });
+              //     const cwtTokenBase64 = base64url.encode(new Uint8Array(cwtTokenBuf));
+              //     request.setVariable('PMUSER_RENEWED_CAT', cwtTokenBase64);
+              //     request.setVariable('PMUSER_RENEWAL_TYPE', renewalType);
+              //   }
+              // }
+              // Proceed and return the content
+              // request.route({ query: finalQs })
```

Comment:
Avoid large chunks of commented code.  This clutters the example and adds difficulty to comprehension of the example code

Reply:
Removed the commented section of the code.